src/spam-protection/payment-human-verification-handler.ts

```diff
@@ -0,0 +1,66 @@
+import { switchMap, take } from 'rxjs/operators';
+
+import { RequestError } from '../common/error/errors';
+import { PaymentAdditionalAction } from '../payment';
+
+import { CardingProtectionChallengeNotCompletedError, CardingProtectionFailedError, SpamProtectionChallengeNotCompletedError } from './errors';
+import GoogleRecaptcha from './google-recaptcha';
+
+export default class PaymentHumanVerificationHandler {
+    private _googleRecaptcha: GoogleRecaptcha;
+
+    constructor(googleRecaptcha: GoogleRecaptcha) {
```

Comment:
Usually we just use the shorthand, i.e.:

```ts
constructor(
    private _googleRecaptcha: GoogleRecaptcha
) {}
```

So you don't need to initialise the properties separately in the constructor.